src/Symfony/Bridge/PhpAmqp/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "symfony/php-amqp-bridge",
+    "type": "symfony-bridge",
+    "description": "Symfony PhpAmqp Bridge",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Samuel Rozé",
```

Comment:
If your surname is spelled `Rozé` ... you need to update the PHPdoc of all classes where you spell it as `Roze` 😄

Reply:
Yeah... ASCII destroyed my surname ages ago. I gave up on the `é` so will revert to `e` 😛